src/commands/Utility/minecraft.ts

```diff
@@ -133,14 +133,13 @@ const command: Command = {
 						color: colors.success,
 						author: { name: getString("moduleName") },
 						title: getString("history.nameHistoryFor", { variables: { username } }),
-						description:
-							nameHistory.length - 1
-								? nameHistory.length - 1 === 1
-									? getString(isOwnUser ? "history.youChangedName1" : "history.userChangedName1", { variables: { username } })
-									: getString(isOwnUser ? "history.youChangedName" : "history.userChangedName", {
-											variables: { username, number: nameHistory.length - 1 },
-									  })
-								: getString(isOwnUser ? "history.youNeverChanged" : "history.userNeverChanged", { variables: { username } }),
+						description: isOwnUser
+							? getString("history.youChangedName", {
+									variables: { number: nameHistory.length - 1 },
+							  })
+							: getString("history.userChangedName", {
```

Comment:
you can simplify this by putting the ternary in the path parameter instead of two getString calls